### ISSUE TITLE
Data source

### ISSUE THREAD

**Issue by SimJeg** [SayBender/Nemo#2]
Hello,

I was wondering the exact source of the Alertwildfire data used in your paper. Was it : https://youtube.com/@nvseismolab ?

Thanks for your work,
Simon

**Comment by SayBender** [comment]
Partially, 

For training the model, I used the videos from the YouTube channel your referred to which ALERTWildfire had captured and shared with us separately including 1300+ videos. We then extracted a lot of frames from these videos. I can't confirm that the videos shared with us have more or less that what is in the YouTube channel. But for certain, anything that has been added since 2020 would not be in our source.

For testing, we used a non-overlapping group of frames plus brand new data that I collected from the fires that were raging on as I was working on the experiments. I would go on AlertWildfire website, find cameras close to the regions where fire was currently burning and capture some images from live feeds and use it for testing. We also used HPWREN database for testing and evaluation plus some fire images from the internet. I also manually collected images from the internet to create the collage images and also for the challenging negative test set. For the negative test set, I would also go to AlertWildfire and HPWREN cameras live, and find images containing smoke-like objects that would trick me into believing it is a smoke and use those as challenging negative test set. 

So in short, the training set was from Videos that our collaborators (University of Nevada's NVseismo @ AlertWildfire) had shared with us that included the YouTube videos. 
But the validation/test sets were from variety of sources to make it challenging. 

I hope I have answered the question and thus, be able to mark this issue as solved.

**Comment by SimJeg** [comment]
Thanks for this very detailed answer :)
